cg/cli/workflow/taxprofiler/base.py

```diff
@@ -138,3 +136,56 @@ def run(
         raise click.Abort() from error
     if not dry_run and not use_nextflow:
         analysis_api.add_pending_trailblazer_analysis(case_id=case_id)
+
+
+@taxprofiler.command("start")
+@ARGUMENT_CASE_ID
+@OPTION_LOG
+@OPTION_WORKDIR
+@OPTION_PROFILE
+@OPTION_CONFIG
+@OPTION_PARAMS_FILE
+@OPTION_REVISION
+@OPTION_COMPUTE_ENV
+@OPTION_USE_NEXTFLOW
+@OPTION_TOWER_RUN_ID
+@DRY_RUN
+@click.pass_context
+def start(
+    context: CGConfig,
+    case_id: str,
+    log: str,
+    work_dir: str,
+    profile: str,
+    config: str,
+    params_file: str,
+    revision: str,
+    compute_env: str,
+    use_nextflow: bool,
+    nf_tower_id: Optional[str],
+    dry_run: bool,
+) -> None:
+    """Start full workflow for CASE ID."""
+    LOG.info(f"Starting analysis for {case_id}")
+
+    try:
+        context.invoke(resolve_compression, case_id=case_id, dry_run=dry_run)
+    except DecompressionNeededError as error:
+        LOG.error(error)
+        raise click.Abort() from error
+    context.invoke(config_case, case_id=case_id, dry_run=dry_run)
+    context.invoke(
+        run,
+        case_id=case_id,
+        log=log,
+        work_dir=work_dir,
+        from_start=True,
+        profile=profile,
+        nf_tower_id=nf_tower_id,
+        config=config,
+        params_file=params_file,
+        revision=revision,
+        compute_env=compute_env,
+        use_nextflow=use_nextflow,
+        dry_run=dry_run,
+    )
```

Comment:
I am thinking if I should also make sure here that if I run the command with nextflow then trailblazer does not update any analysis.

Reply:
that is already part of the run command, so you don't need to repeat that here. We are not updating anything here. Only adding a pending case (which happens within the run command)

Reply:
Right, thanks!